pkg/proto/agentbaker/v1/config.proto

```diff
@@ -8,56 +8,58 @@ message Configuration {
   
   string cluster_certificate_authority = 1;
 
-  // only required until Secure TLS bootstrapping in place
-  // would use kubelet identity after that.
-  string tls_bootstrap_token = 2;
-
   // cluster/user config
-  string kubernetes_version = 3; // Q: can this be auto-detected? Or is this part of specifying the desired node version?
+  string kubernetes_version = 2; // Q: can this be auto-detected? Or is this part of specifying the desired node version?
     // removed kubernetesHyperkubeSpec
-  string kube_binary_url = 4;      // maybe we can combine this and custom_kube_binary_url later once we figure out the logic of how they are used.
```

Comment:
is kube binary URL today used as a default where custom kube binary URL overrides it?